features/step_definitions/integration_steps.rb

```diff
@@ -873,6 +873,14 @@ def non_dc_office_location
   end
 end
 
+And(/^.+ clicks Go To Plans link$/) do
```

Comment:
are we hard-coding english text into a test?

Reply:
Ok good point I found the file where its rendered and used the translation as a link clicking reference and it passes

Reply:
Fixed